xml/System.Collections.Generic/SortedSet`1.xml

```diff
@@ -1739,7 +1739,7 @@
 ## Remarks  
  Any duplicate elements in `other` are ignored.  
   
- If the `other` parameter is a <xref:System.Collections.Generic.SortedSet%601> collection with the same equality comparer as the current <xref:System.Collections.Generic.SortedSet%601> object, this method is an O(n log `m`) operation. Otherwise, this method is an O(`n` log `m`) + O(`n` log `n`) operation, where `n` is the number of elements in `other` and `m` is <xref:System.Collections.Generic.SortedSet%601.Count%2A>.  
+ If the `other` parameter is a <xref:System.Collections.Generic.SortedSet%601> collection with the same equality comparer as the current <xref:System.Collections.Generic.SortedSet%601> object, this method is an `O(n log m)` operation. Otherwise, this method is an `O(n log m) + O(n log n)` operation, where `n` is the number of elements in `other` and `m` is <xref:System.Collections.Generic.SortedSet%601.Count%2A>.  
```

Comment:
I may need to brush up on my BIg O Notation but shouldn't this `O(n log m) + O(n log n)` be this instead `O((n log m) + (n log n))`.

Reply:
I'm looking this up as well. I think you are right, but I want to verify.